recipes-bsp/rpi-eeprom/rpi-eeprom_git.bb

```diff
@@ -0,0 +1,71 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+DESCRIPTION = "This repository contains the rpi4 bootloader and scripts \
+for updating it in the spi eeprom"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=7dcd1a1eb18ae569857c21cae81347cb"
+
+# EEPROM configuration default values for this version, as specified at
+# https://www.raspberrypi.org/documentation/hardware/raspberrypi/booteeprom.md
+SRC_URI = " \
+    git://github.com/${SRCFORK}/rpi-eeprom.git;protocol=git;branch=${SRCBRANCH} \
+    file://0001-rpi-eeprom-config-use-usr-bin-python.patch \
+"
+
+COMPATIBLE_MACHINE = "raspberrypi4-64"
+
+DEPENDS = "python3-native"
```

Comment:
Inheriting `python3native` would be better. But is this dependency actually needed for the build?

EDIT: actually you do it later. So this is not needed anymore.